### PR TITLE
Add network configuration plugin

### PR DESCRIPTION
**Pull request by germanferrero**
This PR adds a plugin which allows to setup the shared password from the limeapp.
In the near future we will be adding more functionalities related to network wise configurations.

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -2,5 +2,5 @@ import { configure } from '@storybook/preact';
 
 configure([
     require.context('../stories', true, /\.stories\.js$/),
-    require.context('../plugins', true, /\.stories\.js$/)
+    require.context('../plugins', true, /\.stories\.js$/),
```
Comment (nicopace) [inline .storybook/config.js]: unneded change. could be removed.

File: plugins/lime-plugin-fbw/src/containers/NetworkForm.js [hunk 2]
```diff
@@ -36,20 +27,18 @@ export const NetworkForm = ({ createNetwork, toggleForm }) => {
 		return e;
 	}
 
-	function _changeHostName(e) {
-		const end = e.type === 'change';
-		e.target.value = slugify(e.target.value, end);
-		setState({ ...state, hostName: e.target.value || '' });
-		return e;
-	}
-
```
Comment (nicopace) [inline plugins/lime-plugin-fbw/src/containers/NetworkForm.js]: this function seems to have been moved untouched just a few lines down.
Would leave it were it is to simplify the commit.

**Review by nicopace — APPROVED** [review]
Made a few comments, but my eyes are not used to React as to other frameworks.
As you have tested it I will unlock the merge.
Would be good to be able to automate each PR with integration so we can test on storybook ;)

good work!

**Comment by germanferrero** [comment]
Thanks for reviewing @nicopace, I have addressed your comments in the last commit :)

**Review by spiccinini — COMMENTED** [review]
Tested!

**Pull request merged**